server/offenceCodeDecisions/Code.ts

```diff
@@ -1,16 +1,20 @@
-export default class Code {
-  code: string
+export class Code {
```

Comment:
This is our unique code to identity exactly which questions have been answered

Reply:
Its basically going to be 100 * paragraph number + the position it appears in the questions - which should leave use with quite a bit of space to put new ones in etc.